README.adoc

```diff
@@ -496,6 +496,20 @@ $ heat action-check stack_name
 $ heat stack-update <parameters> -P node_count=<desired_count>
 ```
 
+== Customize OpenShift installation ==
+
+Those Heat templates make use of openshift-ansible to deploy OpenShift.
+You can provide additional parameters to openshift-ansible by specifying
+a JSON string as the `extra_openshift_ansible_params` parameter.
+For example :
+
+```bash
+$ heat stack-create <parameters> -P extra_openshift_ansible_params="{'osm_use_cockpit':true}"
```

Comment:
I tried with the example above, but json merge fails on this because it doesn't like single quote in "{'osm_use_cockpit':true}" - it needs to be: '{"osm_use_cockpit":true}'